src/style.css

```diff
@@ -1 +1,27 @@
-/* add styles here */
+body {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Consider using the universal selector `*` to reset `box-sizing`, `margin`, and `padding` for all elements. This can help ensure consistent styling across different browsers.